pandas_datareader/oandarest.py

```diff
@@ -25,24 +25,78 @@ class OANDARestHistoricalInstrumentReader(_BaseReader):
             Date to begin fetching curerncy pair, in RFC3339 ("%Y-%m-%dT%H:%M:%SZ)  # Eg: "2014-03-21T17:41:00Z"
         end: string
             Date to end fetching curerncy pair, in RFC3339 ("%Y-%m-%dT%H:%M:%SZ)  # Eg: "2014-03-21T17:41:00Z"
-        granularity: string
-            Time range for each candlestick
-            Default: See DEFAULT_GRANULARITY
+        freq: string
+            Frequency or periodicity of the candlesticks to be retrieved
+            Valid values are the following Panda's Offset Aliases (http://pandas.pydata.org/pandas-docs/stable/timeseries.html):
+                    5S  ->  5 second candlesticks, minute alignment
+                    10S -> 10 second candlesticks, minute alignment
+                    15S -> 15 second candlesticks, minute alignment
+                    30S -> 30 second candlesticks, minute alignment
+                    1T,1min  ->  1 minute candlesticks, minute alignment
+                    2T,2min  ->  2 minute candlesticks, hour alignment
+                    4T,4min  ->  4 minute candlesticks, hour alignment
+                    5T,5min  ->  5 minute candlesticks, hour alignment
+                    15T,15min -> 15 minute candlesticks, hour alignment
+                    30T,30min -> 30 minute candlesticks, hour alignment
+                    1H  ->  1 hour candlesticks, hour alignment
+                    2H  ->  1 hour candlesticks, day alignment
+                    3H  ->  3 hour candlesticks, day alignment
+                    4H  ->  4 hour candlesticks, day alignment
+                    6H  ->  6 hour candlesticks, day alignment
+                    8H  ->  8 hour candlesticks, day alignment
+                    12H -> 12 hour candlesticks, day alignment
+                    1D  ->  1 day candlesticks, day alignment
+                    1W  ->  1 week candlesticks, aligned to start of week
+                    1M  ->  1 month candlesticks, aligned to first day of the month
+                    See OANDA REST v20 for updated list
+            Default: See DEFAULT_FREQUENCY
         candleFormat: string
             Candlesticks representation
             Valid values: M,B,A
+                M for midpoint
+                B for Bid
+                A for Ask
             Default: See DEFAULT_CANDLE_FORMAT
         access_credential: Dict of strings
             Credential to query the api
             credential["accountType"]="practise". Mandatory. Valid values: practice, live
             credential["apiToken"]="Your OANDA API token". Mandatory. Valid value: See your OANDA Account's API Token
     """
-    DEFAULT_GRANULARITY="S5"
+    DEFAULT_FREQUENCY="5S"
     DEFAULT_CANDLE_FORMAT="M"
+    SUPPORTED_OFFSET_ALIASES = {
+            "5S":"S5",
+            "10S":"S10" ,
+            "15S":"S15",
+            "30S":"S30",
+            "1T":"M1",
+            "1min":"M1",
+            "2T":"M2",
+            "2min":"M2",
```

Comment:
When you define `SUPPORTED_OFFSET_ALIASES`

You should have Pandas Offset as key not string shortcut as key.
Because it urge you to repeat several times some values.

```python
In [1]: from pandas.tseries.frequencies import to_offset

In [2]: to_offset("1Min")
Out[2]: <Minute>

In [3]: offset = to_offset("1Min")

In [4]: offset
Out[4]: <Minute>

In [5]: type(offset)
Out[5]: pandas.tseries.offsets.Minute
```